debian/openstudioapp.desktop

```diff
@@ -9,4 +9,4 @@ Icon=openstudioapp
 Terminal=false
 Categories=Utility;X-OpenStudio;X-EnergyModeling;
 MimeType=application/x-openstudio;
-
+StartupWMClass=OpenStudioApp
```

Comment:
I think that's what's missing for 24.04, but not sure.

I tested that it still works on 20.04 with it though

Reply:
https://askubuntu.com/questions/1516383/missing-icons-in-dock-after-upgrade-to-ubuntu-24-04

Reply:
```
xprop WM_CLASS
```